XenModel/Actions/Updates/DownloadAndUnzipXenServerPatchAction.cs

```diff
@@ -209,15 +209,16 @@ private void ExtractFile()
             if (string.IsNullOrEmpty(PatchPath) && downloadUpdate)
             {
                 MarkCompleted(new Exception(Messages.DOWNLOAD_AND_EXTRACT_ACTION_FILE_NOT_FOUND));
-                log.DebugFormat("File '{0}.{1}' could not be located in downloaded archive", updateName, updateFileExtensions);
+                log.InfoFormat("No '{0}' file could be located in downloaded archive", string.Concat(updateFileExtensions));

```

Comment:
It might be worth keeping the update name as well so it's easier to find it in a big log file with many similar failed actions; for example, could we write something like:
```
log.InfoFormat("The downloaded archive does not contain a file with any of the following names: '{0}'",
    string.Join(", ", extensions.Select(s => string.Format("{0}.{1}", update, s))));
```

Reply:
We are not really looking for a particular update in the archive, we take the first one we find with a matching extension. That's why I thought that saying ""File '{0}.{1}' could not be located..." was not quite accurate.

Reply:
Ah ok, fair enough. But I would still join the strings instead of concatenating them, it makes for a prettier log line.